### ISSUE TITLE
Questions regarding phase 1 Duke and user guide

### ISSUE THREAD

**Issue by Riyas97** [nusCS2113-AY1920S1/forum#66]
Hi. I have 2 questions with regards to the Duke application that we did in phase 1 and the user guide that we are supposed to submit this week. To my best of knowledge, I don't think these questions have been asked yet. But, if these questions had been asked before, I will appreciate if someone can answer these questions. Thank you.

1) Our group is intending to morph the existing Duke for phase 2 and there are some parts of the existing Duke application that we will not be using for phase 2 (such as creating a deadline task and etc). As such, is it okay to delete away these codes? Or will there be a situation where we will need these codes in the upcoming weeks?

2) Can we still do changes to our user guide even after the submission this week? This is because my group fears that we might not be able to complete all the features stated in the user guide due to time constraint. As such, we want to know whether can we still change our user guide or will have to stick with the submitted user guide no matter what.

Thank you once again.

**Comment by okkhoy** [comment]
1. Yes, you can remove such code that doesn't apply to your product. (However, do it incrementally, not delete all at once). If you need the same code later, you can use version control to retrieve it. 
2. Yes! This is an initial draft only, we will give you feedback on the UG, particularly if we find that you are planning too much, we will warn you! Your UG should contain everything that you envision in your product. It is not necessary to implement everything. 
    - UG serves 2 purposes: (i) Allows you to think what all should go into the product to make it the most awesome product! (ii) Allows us to evaluate how well you can write documentation. 
    - In light of (ii) above, you should not remove things not implemented in the project. Instead, mark them all as (`Coming in V2.0`).

**Comment by Riyas97** [comment]
Ok noted. Thank you so much.

**Comment by okkhoy** [comment]
Reopening for information to all

**Comment by Lucria** [comment]
@okkhoy Sorry for riding on this issue, but in light of the user guide, is the submission for our developer guide an initial draft as well?

**Comment by okkhoy** [comment]
Yes, both documents are drafts. You can (and should?) update them as you progress in your project.

**Comment by Lucria** [comment]
@okkhoy Thank you very much for your clarification!
